test/JDBC/expected/bbf_view_def.out

```diff
@@ -0,0 +1,308 @@
+master#!#dbo#!#def_v2#!#create view  def_v2<newline>as<newline>select 							2 as a;
+master#!#def_sch1#!#def_v1#!#create view def_sch1.def_v1 as select 1;
+~~END~~
+
+
+create database db1;
+go
+
+use db1;
+go
+
+create view db1_v1 as select 1;
+go
+
+create schema db1_sch1;
+go
+
+create view db1_sch1.db1_v2 as select 2;
+go
+
+select sb.name, vd.schema_name, vd.object_name, vd.definition from babelfish_view_def vd left join sys.sysdatabases sb on vd.dbid=sb.dbid order by vd.dbid, vd.schema_name, vd.object_name, vd.definition;
+go
+~~START~~
+text#!#varchar#!#varchar#!#ntext
+master#!#dbo#!#def_v1#!#create view def_v1 as select c1 as a from vd_t1;
+master#!#dbo#!#def_v2#!#create view  def_v2<newline>as<newline>select 							2 as a;
+master#!#def_sch1#!#def_v1#!#create view def_sch1.def_v1 as select 1;
+db1#!#db1_sch1#!#db1_v2#!#create view db1_sch1.db1_v2 as select 2;
+db1#!#dbo#!#db1_v1#!#create view db1_v1 as select 1;
+~~END~~
+
+
+-- CASE: drop schema and database which contains the view
+use master;
+go
+
+-- should be blocked as it has dependent objects
+drop schema def_sch1;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: cannot drop schema master_def_sch1 because other objects depend on it)~~
+
+
+drop database db1;
+go
+
+select sb.name, vd.schema_name, vd.object_name, vd.definition from babelfish_view_def vd left join sys.sysdatabases sb on vd.dbid=sb.dbid order by vd.dbid, vd.schema_name, vd.object_name, vd.definition;
+go
+~~START~~
+text#!#varchar#!#varchar#!#ntext
+master#!#dbo#!#def_v1#!#create view def_v1 as select c1 as a from vd_t1;
+master#!#dbo#!#def_v2#!#create view  def_v2<newline>as<newline>select 							2 as a;
+master#!#def_sch1#!#def_v1#!#create view def_sch1.def_v1 as select 1;
+~~END~~
+
+
+-- CAST: Check for session properties like ANSI_NULLS and QUOTED_IDENTIFIER
+-- Also check is_schema_bound and uses_database_collation columns
+SET ANSI_NULLS ON;
+go
+
+create view view_ansinullon as select 1;
+go
+
+SET ANSI_NULLS OFF;
+go
+
+create view "view_ansinulloff_qidon" as select 1;
+go
+
+SET QUOTED_IDENTIFIER OFF;
+go
+
+create view view_ansinulloff_qidoff as select 1;
+go
+
+select vd.object_name, vd.definition, vd.is_ansi_nulls_on, vd.uses_quoted_identifier, vd.is_schema_bound, vd.uses_database_collation from babelfish_view_def vd left join sys.sysdatabases sb on vd.dbid=sb.dbid where vd.object_name like ('view_ansi%') order by vd.dbid, vd.schema_name, vd.object_name, vd.definition;
+go
+~~START~~
+varchar#!#ntext#!#bit#!#bit#!#bit#!#bit
+view_ansinulloff_qidoff#!#create view view_ansinulloff_qidoff as select 1;#!#0#!#0#!#0#!#0
+view_ansinulloff_qidon#!#create view "view_ansinulloff_qidon" as select 1;#!#0#!#1#!#0#!#0
+view_ansinullon#!#create view view_ansinullon as select 1;#!#1#!#1#!#0#!#0
+~~END~~
+
+
+
+-- TODO: Add support when alter view is supported
+alter view def_v1 as select 2;
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: 'ALTER VIEW' is not currently supported in Babelfish)~~
+
```

Comment:
Let's add a select statement here as well because once we support alter view then the result of the select statement should be changed.